### PR TITLE
Fix #72 - Share Button not clickable

### LINKED ISSUE #72: feat: Implement share app feature in the application [GCI]
**Summary:**

Add the share app feature in the application

**Steps to reproduce:**

- Open the application
- Click on `Share` in the navigation drawer
- Nothing happens

**Expected behaviour:**

The task here is to add this functionality

### PR DESCRIPTION
**Pull request by OussEmaDevCode**
Fixes #72 

Please Add Screenshots If there are any UI changes.
video2gif_20191205_141255

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/DashboardActivity.kt [hunk 1]
```diff
@@ -185,6 +185,12 @@ class DashboardActivity : MifosBaseActivity(), View.OnClickListener, NavigationV
             R.id.item_settings -> {
                 replaceFragment(SettingsFragment.newInstance(), true, R.id.container)
             }
+            R.id.item_share -> {
+                val sharingIntent = Intent(android.content.Intent.ACTION_SEND)
+                sharingIntent.type = "text/plain"
+                sharingIntent.putExtra(android.content.Intent.EXTRA_TEXT, getString(R.string.share_msg))
+                startActivity(Intent.createChooser(sharingIntent, "Share via"))
```
Comment (SaumyaSingh1) [inline app/src/main/kotlin/org/mifos/mobile/cn/ui/mifos/DashboardActivity.kt]: @OussEmaDevCode  use strings.xml for all strings .

**Comment by SaumyaSingh1** [comment]
@OussEmaDevCode  just few more changes .

**Comment by OussEmaDevCode** [comment]
@SaumyaSingh1 @Sanyam96 Done !!

**Comment by rachittt** [comment]
solved in #88

**Pull request closed without merging**